packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html

```diff
@@ -20,20 +20,24 @@
 <p data-translate="hosting_multisite_git_association_configure_description"></p>
 
 <form class="form-inline" name="$ctrl.gitAssociationForm">
-    <h3 data-translate="hosting_multisite_git_association_ssh_key"></h3>
-    <p
-        data-translate="hosting_multisite_git_association_ssh_key_description"
-    ></p>
-    <div class="row">
-        <oui-field
-            class="col-md-4"
-            data-label="{{ ::'hosting_multisite_git_association_ssh_key' | translate }}"
-        >
-            <oui-clipboard
-                name="sshKey"
-                data-model=":: $ctrl.sshKey"
-            ></oui-clipboard>
-        </oui-field>
+    <div
+        data-ng-if="$ctrl.model.repositoryUrl && !$ctrl.model.repositoryUrl.includes('https')"
```

Comment:
What happens if the `repositoryUrl` is for example either  `git://https-module` or `http://...` ?